RiotNet/Models/Summoner.cs

```diff
@@ -14,12 +14,17 @@ public class Summoner
         /// </summary>
         [Key]
         [DatabaseGenerated(DatabaseGeneratedOption.None)]
-        public long Id { get; set; }
+        public string Id { get; set; }
+
+        /// <summary>
+        /// Gets or sets the player universally unique identifier (PUUID)
+        /// </summary>
+        public string PUUID { get; set; }
```

Comment:
Only capitalize the first letter of acronyms (`Puuid`)

Reply:
done